src/test/java/io/jenkins/plugins/miscjenkinsinfotools/generator/FindJobsTests.java

```diff
@@ -5,30 +5,33 @@
 import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition;
 import org.jenkinsci.plugins.workflow.job.WorkflowJob;
 import org.jenkinsci.plugins.workflow.job.WorkflowRun;
-import org.junit.Rule;
-import org.junit.Test;
-import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
 import org.jvnet.hudson.test.JenkinsRule;
+import org.jvnet.hudson.test.junit.jupiter.WithJenkins;
 
-public class FindJobsTests {
+@WithJenkins
+class FindJobsTests {
 
     private final String[] list = {"test-x-0", "test-x-1", "test-x-2"};
 
-    @Rule
-    public JenkinsRule jenkins = new JenkinsRule();
+    private JenkinsRule jenkins;
+
+    @BeforeEach
+    void beforeEach(JenkinsRule rule) {
+        jenkins = rule;
+    }
 
     @Test
-    @Order(1)
```

Comment:
Removing the `@Order` annotation as they are considered bad practice unless they are actually required to build up dependencies between tests. The order of tests is deterministic already to guarantee reproducible results.